lib/src/components/NxFileUpload/NxFileUpload.scss

```diff
@@ -20,7 +20,6 @@
     // focus styles must be reimplemented because it isn't the actual button that gets focus, it's the input
     &:focus + .nx-btn {
       border-color: var(--nx-color-interactive-border-focus);
```

Comment:
Looks like you need to re-implement the tertiary button focus style here. Currently when you tab through, you get the blue border when focused instead.